server/src/main/java/org/opensearch/bootstrap/BootstrapChecks.java

```diff
@@ -774,7 +774,7 @@ public BootstrapCheckResult check(BootstrapContext context) {
                 String.format(
                     Locale.ROOT,
                     "the default discovery settings are unsuitable for production use; at least one of [%s] must be configured",
-                    Stream.of(DISCOVERY_SEED_HOSTS_SETTING, DISCOVERY_SEED_PROVIDERS_SETTING, INITIAL_MASTER_NODES_SETTING)
+                    Stream.of(DISCOVERY_SEED_HOSTS_SETTING, DISCOVERY_SEED_PROVIDERS_SETTING, INITIAL_CLUSTER_MANAGER_NODES_SETTING)
```

Comment:
Nitpick - We're not branching on version here so the message may be misleading

Reply:
Hi @kartg , thanks a lot for your review! 👍 I got your point. I will also revise the other setting name changes in the error message.

Reply:
Could I double check when the message will be misleading, if it contains the new setting name?
The situation that I could think of is: in a cluster with nodes in both 1.x and 2.x versions, so the user can only configure the setting with old name (`initial_master_node`).

Reply:
Thank your so much for the reply! I didn't think of this edge case of a cluster with mixed version nodes, until you reminded me.
After got your reminder, my idea was telling user use a deprecated setting name is better than telling a unsupported setting name, so I keep the old setting name.
Haha, I totally agree that the most appropriate message is `cluster.initial_cluster_manager_nodes / cluster.initial_master_nodes`, let me check the right balance.

Reply:
I added the both old and new setting name into the error message, and separate by slash `/`, in commit https://github.com/opensearch-project/OpenSearch/pull/2463/commits/2ee85dad58cb0b2f7c726e24820fc23e229346f8 😁